src/MailingListBundle/Resources/config/routing_admin.yml

```diff
@@ -12,12 +12,6 @@ opifer_mailing_list_mailing_list_edit:
     requirements:
         id:  \d+
 
-opifer_mailing_list_mailing_list_subscriptions:
-    path: /mailinglist/subscriptions/{id}
```

Comment:
Are you sure you're removing the correct one? Since they both have a different path defined.

Reply:
This one is unused